news/2025/2025-06-24-state-of-the-ranked-section-survey.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: State of the Ranked section survey
```

Comment:
i don't think this works that well

```suggestion
title: The State of Ranked Beatmaps: A Survey
```

```suggestion
title: Survey: The State of Ranked
```

```suggestion
title: The State of Ranked Maps
```

Reply:
Opting to choose the second option. Simplest, most straightforward.